src/components/Editor/TextPreview.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+import TextField from "@mui/material/TextField";
+import React, { useState } from "react";
+import { Box, IconButton } from "@mui/material";
+import ExpandIcon from "@mui/icons-material/Expand";
+
+const Header = styled.div`
+  color: white;
+`;
+
+const TextDiv = styled.div`
+  background: #3f3d56;
+  max-height: 80%;
+  display: flex;
+  flex-direction: column;
+  border-radius: 15px;
+`;
+
+const DefaultButton = styled.div`
+  padding: 0.5em;
+  cursor: pointer;
+  text-align: center;
+  font-family: Inter, monospace;
+  font-size: 1.5em;
+  color: white;
+  background: #3f3d56;
+  max-height: 22%;
+  border-radius: 15px;
+`;
+
+const TextPreview = ({ desc }: any): JSX.Element => {
```

Comment:
The text preview looks weird : 
![image](https://user-images.githubusercontent.com/44596199/158079905-938d0223-0ebf-458d-9bc4-6841f9c04434.png)

Reply:
Now the text box looks like this if the action does not have a text : 
![image](https://user-images.githubusercontent.com/44596199/158084780-7896a832-0a47-4c8a-be44-f4add2f2920c.png)